### ISSUE TITLE
ACC-261.8(Level AA) _  chart _  Not visible in high contrast theme

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3268]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Chart
Affected Check Point:	ACC-261.8
Short text:	ACC-261.8(Level AA) _  chart _  Not visible in high contrast theme

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Select +Upload YAML.
•	Copy the following code and paste it into the editor: 

apiVersion: v1
kind: Namespace
metadata:
  name: nginx-ns
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  namespace: nginx-ns
  labels:
    app: nginx
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
•	Select Upload
•	Select Close
•	Explore the Monitoring and Health section of the Cluster Details page.
•	In the Events section, select All Messages.
•	Select the sorting icon.
•	In the Sort By section, select Last seen.
•	Select OK.

3. Issue Type: Manual
Observed Behavior:
In High contrast white/black theme, The radial chart bars indicators are not visible 
Screenshot:
 
Screenshot 2:
 
Expected Behavior:
n high contrast black theme, the focus indication provided for 'Product' bar chart should be displayed with a contrast ratio of at least 4.5:1.

In high contrast theme graphical objects (like charts) and visual details which identify a UI element and its state that form filled shapes should have minimum contrast ratio of 4.5:1.

A minimum contrast improves legibility for partially sighted users (for example, users with low visual acuity or color vision deficiencies) and makes content easier to use. Therefore, all visual details (such as lines, colors, …) that identify a UI element, show its state or indicate its interoperability need to fulfill a minimum contrast.
[ACC-261.8(Level AA) _  chart _  Not visible in high contrast theme.docx](https://github.com/user-attachments/files/16688096/ACC-261.8.Level.AA._.chart._.Not.visible.in.high.contrast.theme.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16688098/Issue.processing-09.2021.2.docx)
